mayastor/src/bin/casperf.rs

```diff
@@ -0,0 +1,405 @@
+use std::{cell::RefCell, os::raw::c_void, ptr::NonNull};
+
+use clap::{value_t, App, AppSettings, Arg};
+use rand::Rng;
+
+use mayastor::{
+    core::{
+        mayastor_env_stop,
+        Bdev,
+        Cores,
+        Descriptor,
+        DmaBuf,
+        IoChannel,
+        MayastorCliArgs,
+        MayastorEnvironment,
+        Mthread,
+        Reactors,
+    },
+    logger,
+    nexus_uri::bdev_create,
+    subsys::Config,
+};
+use spdk_sys::{
+    spdk_bdev_free_io,
+    spdk_bdev_read,
+    spdk_bdev_write,
+    spdk_poller,
+    spdk_poller_unregister,
+};
+
+#[derive(Debug)]
+enum IoType {
+    /// perform random read operations
+    READ,
+    /// perform random write operations
+    #[allow(dead_code)]
```

Comment:
`Write` seems to be used, is this dead_code allow needed?

Reply:
Ha! -- you cant fool the compiler =) "used" means actually used not just typed in the code somewhere, so no, it will complain :)